indra/newview/skins/default/xui/en/menu_gallery_inventory.xml

```diff
@@ -125,6 +125,22 @@
          function="Inventory.DoToSelected"
          parameter="thumbnail" />
     </menu_item_call>
+    <menu_item_call
+     label="Add to favorites"
+     layout="topleft"
+     name="favorites_add">
+        <menu_item_call.on_click
+         function="Inventory.DoToSelected"
+         parameter="favorite" />
+    </menu_item_call>
+  <menu_item_call
+   label="Remove from favorites"
+   layout="topleft"
+   name="favorites_remove">
+    <menu_item_call.on_click
+     function="Inventory.DoToSelected"
+     parameter="favorite" />
+  </menu_item_call>
```

Comment:
Do we need these 2 menu items? We have similar 2 below.

Reply:
Removed, thank you.